schema.d.ts

```diff
@@ -22,6 +22,11 @@
  * canonical export that should be used.
  */
 export interface Package {
+  /**
+   * The name of the schema, it's best to be unique.
```

Comment:
This should say that it's the name of the package.

We almost certainly want to namespace the package names too, since npm isn't the only package registry in town. We have at least GitHub, JSR, and internal repositories. We should discuss namespacing conventions in #119